### PR TITLE
Addresses #1908 to create an upper cell type slim

### PR DESCRIPTION
**Pull request by aleixpuigb**
Addresses #1908 to create an upper cell type slim

### CODE REVIEW

File: docs/odk-workflows/RepositoryFileStructure.md [hunk 1]
```diff
@@ -39,3 +39,4 @@ These are the components in CL
 | mappings.owl | None |
 | blood_and_immune_upper_slim.owl | None |
 | eye_upper_slim.owl | None |
+| cell_type_upper_slim.owl | None |
```
Comment (bvarner-ebi) [inline docs/odk-workflows/RepositoryFileStructure.md:42]: Consider renaming to **general_cell_type_upper_slim.owl**

**Comment by aleixpuigb** [comment]
I have tested the coverage for `animal cell` (CL:0000548), and it is 90%, so not bad (Full report [here](https://docs.google.com/spreadsheets/d/1jNLndoBK0RKXUCrI9-5BZAO35rUBjmTKlL7LSK-1xwM/edit?usp=sharing):



#####Coverage percentage##### |  
-- | --
90.46% |  
#####Number of terms covered by each term in the slim##### |  
exocrine cell | 23
bone cell | 39
smooth muscle cell | 59
striated muscle cell | 75
endocrine cell | 89
stem cell | 103
connective tissue cell | 206
neural cell | 595
hematopoietic cell | 659
epithelial cell | 721

**Review by dosumis — APPROVED** [review]
I think I'd like something a bit more granular - maybe a bit closer to the original suggestions from chat GPT. Also - there's some work to do with the terms not covered - finding covering classes &/or re-arranging hierarchy to include them.  However, I think it is important to get something out this month so we can work with it in other projects (enrichment on pandasaurus needs some general classes with good coverage to top out at).  So, I will approve, but we should start another round of work straight afterwards.

**Comment by aleixpuigb** [comment]
>I think I'd like something a bit more granular

That is closer to my first proposal https://github.com/obophenotype/cell-ontology/issues/1908#issuecomment-1545658060, and then we decided to go to a higher level. I can use that one with some edits discussed with @bvarner-ebi.

**Comment by aleixpuigb** [comment]
[This list](https://docs.google.com/spreadsheets/d/1jNLndoBK0RKXUCrI9-5BZAO35rUBjmTKlL7LSK-1xwM/edit#gid=566277485) adds some granularity and still has a coverage above 83%:


#####Coverage percentage##### |  
-- | --
83.37% |  
#####Number of terms covered by each term in the slim##### |  
fat cell | 12
melanocyte | 15
monocyte | 20
exocrine cell | 23
natural killer cell | 28
extraembryonic cell | 29
cell of skeletal muscle | 29
germ line cell | 33
bone cell | 39
glial cell | 41
blood cell | 42
stromal cell | 52
macrophage | 55
smooth muscle cell | 59
retinal cell | 74
dendritic cell | 77
fibroblast | 83
endocrine cell | 89
B cell | 94
stem cell | 103
T cell | 156
neuron | 493
epithelial cell | 721

More work can be done to improve it, but it might be a good first set. Would you be happy to use this subset instead of the previous more general one @dosumis ?

**Pull request merged**